docs/designers-developers/developers/data/data-core-blocks.md

```diff
@@ -195,6 +207,10 @@ _Returns_
 
 <!-- START TOKEN(Autogenerated actions) -->
 
+<a name="addBlockCollection" href="#addBlockCollection">#</a> **addBlockCollection**
+
+Undocumented declaration.
```

Comment:
Documentation is missing here.

Reply:
I added it, I guess this will update automatically?

Reply:
`npm run docs:build` should update this document.